config/routes.rb

```diff
@@ -240,8 +240,13 @@
     get '/manage/email/confirm_delete/:id' => 'users/emails#confirm_delete',
         as: :manage_email_confirm_delete
 
-    get '/add/phone' => 'users/phones#add'
-    post '/add/phone' => 'users/phones#create'
+    if IdentityConfig.store.consolidate_phone_controllers
+      get '/add/phone' => 'users/phone_setup#index'
+      post '/add/phone' => 'users/phone_setup#create'
+    else
+      get '/add/phone' => 'users/phones#add'
+      post '/add/phone' => 'users/phones#create'
+    end
```

Comment:
Is there any way we could make the new controller have new URLs? I'm assuming the form fields will be slightly different between the two controllers, so that makes for a hectic period during a deploy, having a separate controller might help make for a smoother transition?

Reply:
Its not a new controller its allowing users to redirect to a common controller for account creation and adding new phone number.

Reply:
Will this work if, in the 50/50 state, somewhere were to land at the old controller page, but then submit the values from that page to the new controller `POST` endpoint?